torch/_tensor.py

```diff
@@ -136,6 +136,16 @@ def _reduce_ex_internal(self, proto):
                        str(self.device),
                        self.requires_grad)
             return (torch._utils._rebuild_mlc_tensor, arg_mlc)
+        if self.device.type == 'meta':
+            # NB: This implementation BREAKS storage sharing.  Current
```

Comment:
Meta Tensors actually have storage?

Reply:
Yes they do. I added this so that when running tests with meta tensors I could reproduce memory overlap errors.